client/templates/typescript-fetch-templates/runtime.mustache

```diff
@@ -0,0 +1,344 @@
+/* tslint:disable */
```

Comment:
I have mixed feelings about copying all the `runtime.ts` content here for a slightly better error handling 🤔

Reply:
I agree it's not great to copy everything just for that. I haven't found a better way. For some reason, the original runtime throws a fetch's Response object, not an Error. 

https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/typescript-fetch/runtime.mustache#L42

This brings issues when doing exception handling. Each client needs to process it correctly which is not trivial. The response object doesn't have an HTTP response body, it needs to be pulled. Rxjs and node are not super happy when raising other objects that Error.